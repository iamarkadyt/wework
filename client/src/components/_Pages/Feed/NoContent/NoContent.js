import React from 'react'

const noContent = () => (
  <p className="Feed-message" style={{ marginBottom: 0 }}>
    We've got no posts for you yet!
  </p>
)

export default noContent
