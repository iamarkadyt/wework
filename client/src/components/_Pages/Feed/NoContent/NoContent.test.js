import React from 'react'
import NoContent from './NoContent'
import { shallow } from 'enzyme'
import cloneDeep from 'lodash.clonedeep'

describe('NoContent', () => {
  let mountedComponent, props

  const getMockProps = () => {
    return cloneDeep({})
  }

  const comp = () => {
    if (!mountedComponent) {
      mountedComponent = shallow(<NoContent {...props} />)
    } 
    return mountedComponent
  }

  beforeEach(() => {
    mountedComponent = undefined
    props = getMockProps()
  })

  it('matches snapshot', () => {
    expect(comp()).toMatchSnapshot()
  })
})

