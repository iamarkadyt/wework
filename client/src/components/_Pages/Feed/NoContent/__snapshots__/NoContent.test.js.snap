// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoContent matches snapshot 1`] = `
<p
  className="Feed-message"
  style={
    Object {
      "marginBottom": 0,
    }
  }
>
  We've got no posts for you yet!
</p>
`;
